lib/vty.c

```diff
@@ -346,17 +346,8 @@ int vty_out(struct vty *vty, const char *format, ...)
 	case VTY_SHELL_SERV:
 	case VTY_FILE:
 	default:
-		vty->vty_buf_size_accumulated += strlen(filtered);
```

Comment:
can we do something more limited, just for instance removing the direct call to "vtysh_flush" ? We might well want to keep the counters that are part of the progressive send logic, for example.